src/ex1/script.js

```diff
@@ -0,0 +1,165 @@
+const allTodosList = document.getElementById("all-todos-list");
+
+const unsorted = 0;
+const sortedAsc = 1;
+const sortedDesc = 2;
```

Comment:
such consts are usually named with an uppercase for clarity

Reply:
fixed